src/components/listitems/ListItemTransaction.tsx

```diff
@@ -115,7 +115,9 @@ export const ListItemTransaction = ({
           </View>
         )}
         <View style={IOStyles.flex}>
-          <H6 color={theme["textBody-default"]}>{title}</H6>
+          <H5 numberOfLines={2} color={theme["textBody-default"]}>
```

Comment:
According to the design guidelines, `H5` is a typographic style that renders the text in uppercase. From what I see on the design project file, the correct typographic style is "_LabelSmall (Readex)_". Can you please check out?

(For reference, see also this PR with the new changes suggested: https://github.com/pagopa/io-app-design-system/pull/48)

Reply:
Addressed in 584f43422485a8d6d1a2ef3e1a81a58b6cb71795